package.json

```diff
@@ -24,6 +24,7 @@
     "karma-spec-reporter": "0.0.26",
     "load-grunt-config": "0.19.2",
     "phantomjs-prebuilt": "2.1.7",
+    "redux-form": "^7.4.2",
```

Comment:
Just as a head up on this one to remove.

Reply:
Thanks for the reminder!